### PR TITLE
Add boomr config

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Add boomr config provided my mpulse

**Code changes:**

- Add boomr config

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)~
- ~[ ] This PR requires manual testing~

### CODE REVIEW

File: src/app/containers/MPulseBeacon/boomr.js [hunk 1]
```diff
@@ -9,6 +9,14 @@ const boomr = (apiKey) => {
   if (window.BOOMR && (window.BOOMR.version || window.BOOMR.snippetExecuted)) {
       return;
   }
+  window.BOOMR_config = {
+    Early: {
+      enabled: true,
```
Comment (chris-hinds) [inline src/app/containers/MPulseBeacon/boomr.js:14]: @dr3 what is `enabled: true` doing here?

Comment (dr3) [reply src/app/containers/MPulseBeacon/boomr.js:14]: > drew Today at 8:12 AM
> The domain we missed to add to our csp is https://*.akamaihd.net which isnt listed in the CSP section of https://developer.akamai.com/tools/boomerang :disappointed:
> 
> Simon Hearne  10 minutes ago
> Is the akamai plugin disabled?
> 
> drew  9 minutes ago
> Not sure what that is soz :stuck_out_tongue:
> 
> drew  8 minutes ago
> we run > https://github.com/bbc/simorgh/blob/latest/src/app/containers/MPulseBeacon/boomr.js
> 
> Simon Hearne  8 minutes ago
> ```
> window.BOOMR_config = {
> Early: {
>     enabled: true
>     },
> Akamai: {
>     enabled: false
>   }
> };
> ```
> 
> drew  7 minutes ago
> Should we add that?
> 
> Simon Hearne  7 minutes ago
> Yep, that will prevent those requests to akamaihd.net

**Review by chris-hinds — APPROVED** [review]

**Review by FK78 — APPROVED** [review]

**Review by thekp — APPROVED** [review]
you owe us a nandos

**Comment by dr3** [comment]
YOLO, will fix forward, only affects test

**Pull request merged**